lib/bolt/config/options.rb

```diff
@@ -233,8 +233,7 @@ module Options
                        "install` command.",
           type: Array,
           items: {
-            type: Hash,
-            required: ["name"],
```

Comment:
We should keep `required`, otherwise the schema won't flag hashes that are missing it.